### PR TITLE
Add the config for remote pipeline

### PR DESCRIPTION
**Pull request by jmuk**
This is meaningful only with https://github.com/googleapis/artman/pull/22

### CODE REVIEW

File: gapic/api/artman_common.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+common:
```
Comment (garrettjonesgoogle) [inline gapic/api/artman_common.yaml]: If we can depend on this file, why not get rid of the individual config files with the hardcoded values?

Comment (jmuk) [inline gapic/api/artman_common.yaml]: execute_pipeline.py is a generic utility and does not have good commandline to allow expanding variables like 'API_VERSION' or 'API_NAME'. My new commandline tool, api_codegen.py, has narrower focus and therefore can use this simplified format.

Until execute_pipeline.py is redesigned to allow this config, I think it's better to keep the individual API config files.

**Comment by bjwatson** [comment]
@jmuk Is this PR obsolete?

**Comment by jmuk** [comment]
Right.

**Pull request closed without merging**